packages/http-client-csharp/emitter/src/lib/typespec-server.ts

```diff
@@ -73,7 +74,7 @@ export function resolveServers(
       const variable: InputParameter = {
         Name: "host",
         NameInRequest: "host",
-        Description: server.description,
+        Doc: server.description,
```

Comment:
why the server only has description? should this also be changed to summary and doc?

Reply:
this is from typespec library: https://github.com/microsoft/typespec/blob/1cf8601d0f65f707926d58d56566fb0cb4d4f4ff/packages/http/src/decorators.ts#L464-L468

there is only `description`